src/sections/error-page/ErrorPage.module.scss

```diff
@@ -0,0 +1,30 @@
+@use 'sass:color';
+@import 'node_modules/@iqss/dataverse-design-system/src/lib/assets/styles/design-tokens/colors.module';
+@import 'src/assets/variables';
+
+.section-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  min-height: $main-container-available-height;
```

Comment:
My two cents, min-height should be dynamic based on a prop.
This min-height is fine if we have the presence of the header and footer.
But if the error occurs at the layout level we will have neither header nor footer, causing the container height to not grow to the full height of the viewport, in that case the min-height should be 100vh I believe